rasa/core/test.py

```diff
@@ -488,6 +488,7 @@ def log_failed_stories(failed, out_directory):
     out_directory: Optional[Text] = None,
     fail_on_prediction_errors: bool = False,
     e2e: bool = False,
+    no_plot: bool = False,
```

Comment:
```suggestion
    disable_plotting: bool = False,
```

Reply:
right, I saw that. 
Then name here comes directly from kwargs, thus directly from Argparser. 
I just thought that maybe it's confusing to rename the variable right away using "dest" parameter. But now I think I'll do that.